src/vision.cs

```diff
@@ -173,6 +207,31 @@ interface VNDetectBarcodesRequest {
 		[Protected]
 		[Export ("symbologies", ArgumentSemantic.Copy)]
 		NSString [] WeakSymbologies { get; set; }
+
+		// We must inline the following static properties so the correct class_ptr is used.
```

Comment:
Really minor, but I'd say "the following **4** static properties", because otherwise next year someone will add more properties below and it'll be unclear exactly which properties the comment refers to.